app/index.html

```diff
@@ -42,9 +44,46 @@ <h3 class="index-header">Welcome to the iO Academy aptitude test</h3>
                 </div>
                 <div>
                     <button type="submit" class="start-button">Start test</button>
+                    <button type="button" class="instructions-button" data-toggle="modal" data-target="#instructionsModal">Instructions</button>
                 </div>
             </div>
         </form>
+        <div class="modal fade" id="instructionsModal" tabindex="-1" role="dialog" aria-labelledby="instructionsLabel" aria-hidden="true">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h4 class="modal-title">Instructions</h4>
```

Comment:
missing id instructionsLabel which is present on the other page

Reply:
added